docs/streamtypes/tile-document/api.md

```diff
@@ -57,13 +56,14 @@ The content of your document, in the form of a JSON document. If `schema` is set
 [:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_stream_tile.tilemetadataargs-1.html){:target="_blank"}
 
 
-!!! info "Using the `deterministic` parameter"
-    For most use cases you will likely want to leave the `deterministic` parameter set to false. However for special circumstances, you may want this to be set to true. For example this should be set to true if you would like to enable [deterministic queries](#query-a-deterministic-document) for your document. If this is your use case, then it is also important that you set `content` to `null` during document creation. Deterministic document queries are based entirely on the document's initial metadata. You can proceed to add content to your document by [updating it](#update-a-document).
+!!! warning "Using the `deterministic` parameter"
```

Comment:
this also feels more like info than a warning?

Reply:
same as above